Vagrantfile

```diff
@@ -80,11 +102,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.synced_folder ".", "/home/vagrant/fixmystreet", :owner => "vagrant", :group => "vagrant"
 
-  config.vm.provision "shell" do |s|
-    s.inline = $setup
-    s.args   = "#{baseBox}"
+  # When using the mySociety box, just mount the local perl modules and run `script/update`
+  # For any other box, just run the full setup process.
+  if "#{baseBox}" == "mysociety/fixmystreet"
+    config.vm.provision "shell", run: "always", inline: $mount_modules
+    config.vm.provision "shell", inline: $update
```

Comment:
I'm wondering if this should be always run too; would save you having to do it/remember to do it after logging in.

Reply:
I did wonder about that, in the end I didn't because I thought users might want to decide for themselves whether to run it or not. I don't have a strong opinion, so would be happy to make this change.